Refactored rviz launch file

AdarshKaran opened the pull request:
- Refactored the rviz_launch file to use opaque function
- Added additional launch argument to change the output type (screen, log) of RViz node

AdarshKaran committed:
Refactored rviz launch file

AdarshKaran committed:
Removed contributor tag from package.xml

padhupradheep commented on README.md:
```diff
@@ -4,3 +4,12 @@ Common nav2 bringup package for all the robots
+### **[Date: 2025-02-27]**
+
+- Refactored the rviz_launch file to use opaque function
+- Added additional launch argument to change the output type (screen, log) of RViz node 
```
changelog will be generated by the bloom. Just make sure, your commits are descriptive.

padhupradheep reviewed (changes requested):
LGTM! I see your idea of making it generic for all the robots. This means that we also have to add the rviz_launch.py to the navigation_neo.launch.py. By that way, we just have to add a single argument to our navigation.launch.py across the robots. 

By default, use_rviz should be set to true on the navigation launch. 

Did you have something else in mind?

AdarshKaran commented:
> LGTM! I see your idea of making it generic for all the robots. This means that we also have to add the rviz_launch.py to the navigation_neo.launch.py. By that way, we just have to add a single argument to our navigation.launch.py across the robots.
> 
> By default, use_rviz should be set to true on the navigation launch.
> 
> Did you have something else in mind?

- I have now added the start_rviz node to the navigation_neo.launch.py file and the necessary launch argument.
- I have also updated removed the additional RViz node from navigation.launch.py in ROX and instead added an argument to the navigation_neo.launch.py launch node which already exists. 
- I have removed the changelogs

AdarshKaran committed:
Added the start_rviz node to the navigation_neo.launch.py file and the necessary launch argument, removed changelog

AdarshKaran committed:
Added the missing declare_use_rviz_cmd action in navigation_neo.launch.py

AdarshKaran commented:
- Added the missing `declare_use_rviz_cmd` action in `navigation_neo.launch.py`
